.pre-commit-config.yaml

```diff
@@ -22,7 +22,7 @@ repos:
     rev: v3.0.3
     hooks:
       - id: prettier
-        files: \.(json|yaml|yml)$
```

Comment:
this linter is not compatible with our project's yaml format, so I have disabled it for yaml files. Specifically, it fires for this TTP: 

```
cmd/test-resources/repos/another-repo/some-ttps/cleanup-tests/subttp/ttp.yaml
``` 

and breaks the TTP by messing with the argument references, producing invalid syntax: 

```
@@ -1,7 +1,7 @@
 ---
 name: print-stuff
 args:
-- name: to_print
+  - name: to_print
 steps:
-- name: print-arg
-  print_str: {{ .Args.to_print }}
\ No newline at end of file
+  - name: print-arg
+    print_str: { { .Args.to_print } }
```

We already `yamllint`, so nothing is lost